projects/seamless/index.js

```diff
@@ -44,7 +48,54 @@ async function geyserTvl(api) {
 
 const baseAAVE = aaveExports("base", AAVE_ADDRESSES_PROVIDER_REGISTRY, undefined, [AAVE_POOL_DATA_PROVIDER], { v3: true });
 
-module.exports = mergeExports([{
-  methodology: methodologies.lendingMarket,
-  base: baseAAVE,
-}, { base: { tvl: geyserTvl } }]);
+const SeamlesMorphoVaultsTVL = async (api) => {
```

Comment:
we can create a new sublisting for this, but how do these vaults work? can users borrow against their deposits here? other issue is, even if your team is curating the morpho vaults, the tvl belongs to morpho project. We are thinking of adding curator category to tackle this, but atm, we reject curator projects :(